trl/trainer/orpo_trainer.py

```diff
@@ -68,7 +68,6 @@
 if is_peft_available():
     from peft import PeftModel, get_peft_model, prepare_model_for_kbit_training
 
```

Comment:
```suggestion
if is_liger_kernel_available():
    from liger_kernel.chunked_loss import LigerFusedLinearORPOLoss
```

even if it is not always used, I think it's more readable when all imports are on top. 
+ If there’s an issue with a dependency, it will be detected as soon as the script runs